myhoard/web_server.py

```diff
@@ -133,6 +133,8 @@ async def status_update(self, request):
             elif body_mode == Controller.Mode.observe:
                 self.controller.switch_to_observe_mode()
             elif body_mode == Controller.Mode.restore:
+                if not isinstance(body.get("rebuild_tables"), (bool, type(None))):
```

Comment:
I think the validation is not 100% correct, if the body contains:
```
"rebuild_tables": null
```
the validation will succeed and later the None value will be passed over instead of False.

Reply:
Ah, yes. Will fix that.